src/test/java/com/wcc/platform/controller/MentorshipControllerTest.java

```diff
@@ -124,4 +128,18 @@ void testMentorsOkResponse() throws Exception {
         .andExpect(status().isOk())
         .andExpect(content().json(objectMapper.writeValueAsString(mentorsPage)));
   }
+
+  @Test
+  void testAdHocTimelineOkResponse() throws Exception {
+    var fileName = AD_HOC_TIMELINE.getFileName();
+    var expectedJson = FileUtil.readFileAsString(fileName);
+
+    when(service.getAdHocTimeline()).thenReturn(createMentorshipAdHocTimelinePageTest(fileName));
```

Comment:
You can do this, and it would be the same as the fallback method, right? 

```suggestion
    when(service.getAdHocTimeline()).thenReturn(createMentorshipAdHocTimelinePageTest());
```

Reply:
I've removed the argument.